tests/api/views/register_email/test_register_email.py

```diff
@@ -205,6 +211,49 @@ def test_noverify_ok(self):
         self.user.refresh_from_db()
         self.assertEqual(self.user.email, self.new_email)
 
+    @override_rest_registration_settings({
+        'REGISTER_EMAIL_VERIFICATION_ENABLED': False,
+        'USE_NON_FIELD_ERRORS_KEY_FROM_DRF_SETTINGS': True,
+    })
+    def test_register_email_fail_with_non_field_errors(self):
+        # arrange
+        with mock.patch(
```

Comment:
Not huge fan of mocks, as they have the tendency to get out of sync quickly and lead to strange results. What about using custom user model in the test (like in other test cases - look for `override_auth_model_settings` usage)?